### PR TITLE
Removing 'you' from Spec

### PR DESCRIPTION
**Pull request by j-mike-culhane**
Removed use of 'you' from document and replaced with impersonal expressions.

Signed-off-by: Michael Culhane <j.mike.culhane@gmail.com>

### CODE REVIEW

File: content/shmem_alltoall.tex [hunk 1]
```diff
@@ -103,7 +103,7 @@
     This routine restores \VAR{pSync} to its original contents.  Multiple calls
     to \openshmem\ routines that use the same \VAR{pSync} array do not require
     that \VAR{pSync} be reinitialized after the first call.
-    You must ensure that the \VAR{pSync} array is not being updated by any
+    Ensure that the \VAR{pSync} array is not being updated by any
```
Comment (jdinan) [inline content/shmem_alltoall.tex]: How about "The user must ensure..."?  We use similar phrasing elsewhere in the document.

File: content/shpalloc.tex [hunk 2]
```diff
@@ -25,8 +25,7 @@
     consistency, all \acp{PE} in an program must call \FUNC{SHPALLOC} with the same
     value of length; if any  \acp{PE} are missing, the program will hang.
     
-    By using the \Fortran \CONST{POINTER} mechanism in the following manner, you
-    can use array \VAR{A} to refer to the block allocated by \FUNC{SHPALLOC}:
+    By using the \Fortran \CONST{POINTER} mechanism in the following manner, array \VAR{A} can be used to refer to the block allocated by \FUNC{SHPALLOC}:
```
Comment (jdinan) [inline content/shpalloc.tex]: Please add a hard wrap to split this line.

**Comment by hppritcha** [comment]
@jdinan Mike's pushed requested changes up.  Could you take a look?

**Comment by jdinan** [comment]
Can I get some more +1's?  :+1:

I'm happy to merge this as a DocEdit if we can get to +4.  @hppritcha @shamisp @manjugv @nspark or anyone else who has a few minutes to review the changes.  Thanks!

**Comment by swpoole** [comment]
+1

File: content/shmem_atomic_add.tex [hunk 3]
```diff
@@ -44,7 +43,7 @@
  }
 
 \apidesctable{
-    If you are using \Fortran, \VAR{dest} must be of the following type:
+    When \Fortran, \VAR{dest} must be of the following type:
```
Comment (BryantLam) [inline content/shmem_atomic_add.tex]: +"using"

File: content/shmem_iget.tex [hunk 4]
```diff
@@ -39,19 +39,19 @@
     \apiargument{IN}{dst}{The stride between consecutive elements of the \dest{}
         array.  The stride is scaled by the element size of the \dest{} array.
         A  value of \CONST{1} indicates contiguous data. \VAR{dst} must be of
-        type \textit{ptrdiff\_t}.  If you are calling  from  \Fortran,  it  must
+        type \textit{ptrdiff\_t}.  When calling  from  \Fortran,  it  must
```
Comment (BryantLam) [inline content/shmem_iget.tex]: Consistency: "calling from" -> "using". And same for the change below this one.

File: content/shmem_iput.tex [hunk 5]
```diff
@@ -40,18 +40,18 @@
...
+        of type \textit{ptrdiff\_t}.  When using \Fortran, it must be a
         default integer value.}
     \apiargument{IN}{nelems}{Number of elements in the \dest{} and \source{}
-        arrays.  \VAR{nelems} must be of type \VAR{size\_t} for \Cstd.  If you are
+        arrays.  \VAR{nelems} must be of type \VAR{size\_t} for \Cstd.  When
         using \Fortran, it must be  a constant, variable, or array element of
         default integer type.}
     \apiargument{IN}{pe}{\ac{PE} number of the remote \ac{PE}.  \VAR{pe} must be
-        of type integer.   If you  are  using  \Fortran, it must be a constant,
+        of type integer.   When  \Fortran, it must be a constant,
```
Comment (BryantLam) [inline content/shmem_iput.tex]: +"using"

File: content/shmem_broadcast.tex [hunk 6]
```diff
@@ -67,8 +66,8 @@
     \dest{} and \source{} data objects and the same \VAR{pSync} work array must be
     passed to all \acp{PE} in the \activeset.
     
-    Before any \ac{PE} calls a broadcast routine, you must ensure that the following
-    conditions exist (synchronization via a barrier or some other method is often
+    Before any \ac{PE} calls a broadcast routine, the following
+    conditions must be met: (synchronization via a barrier or some other method is often
```
Comment (BryantLam) [inline content/shmem_broadcast.tex]: (Ref.1)

File: content/shmem_reductions.tex [hunk 7]
```diff
@@ -200,8 +200,8 @@
     and \source{} arrays, and the same \VAR{pWrk} and \VAR{pSync} work arrays, must
     be passed to all \acp{PE} in the \activeset.
     
-    Before any \ac{PE} calls a reduction routine, you must ensure that the
-    following conditions exist (synchronization via a \OPR{barrier} or some other
+    Before any \ac{PE} calls a reduction routine, the
+    following conditions must be met (synchronization via a \OPR{barrier} or some other
```
Comment (BryantLam) [inline content/shmem_reductions.tex]: Mismatched consistency with other comment (Ref.1) of the text in parentheses. I don't know what the OPR command is used for or whether this phrasing is used in other collectives (e.g., collect, all2all).

Comment (BryantLam) [inline content/shmem_reductions.tex]: Mark as FIXME. This requested change is out of scope for this PR.

File: content/shmem_alltoall.tex [hunk 8]
```diff
@@ -103,7 +103,7 @@
     This routine restores \VAR{pSync} to its original contents.  Multiple calls
     to \openshmem\ routines that use the same \VAR{pSync} array do not require
     that \VAR{pSync} be reinitialized after the first call.
-    You must ensure that the \VAR{pSync} array is not being updated by any
+    The user must ensure that the \VAR{pSync} array is not being updated by any
```
Comment (manjugv) [inline content/shmem_alltoall.tex]: We should consider using a macro for "user". So we can get consistent formatting through out the document.

File: content/shmem_iget.tex [hunk 9]
```diff
@@ -39,19 +39,19 @@
...
         be a default integer value.}
     \apiargument{IN}{sst}{The stride between consecutive elements of the
         \source{} array.  The stride is scaled by the element size of the \source{}
         array.  A  value of \CONST{1} indicates contiguous data.  \VAR{sst} must be
-        of type \textit{ptrdiff\_t}.  If you are calling  from  \Fortran,  it  must
+        of type \textit{ptrdiff\_t}.  When calling  from  \Fortran,  it  must
```
Comment (manjugv) [inline content/shmem_iget.tex]: We could also use Bryant suggestion (calling from -> using) here.

File: content/shmem_alltoall.tex (hunk 8 above)
Comment (jdinan) [inline content/shmem_alltoall.tex]: I'm not sure that a macro would work for this because we multiple phrasings involving "user".  The best solution is probably to allow Mike to make the style consistent and then do our best to follow it.

**Comment by hppritcha** [comment]
@j-mike-culhane is it clear what @jdinan is suggesting?  the '%' is like '#' in python, stuff after the '%' will not show up in the pdf file.

**Comment by j-mike-culhane** [comment]
The appropriate changes have been made to the file. My apologies for the confusion. Interacting with github is still a fairly novel experience for me.

**Comment by jdinan** [comment]
Thanks!  @BryantLam could you please review?

File: content/shmem_iget.tex (hunk 4 above)
Comment (BryantLam) [inline content/shmem_iget.tex]: Consistency: "When calling from" --> "When using"

File: content/shmem_iget.tex (hunk 9 above)
Comment (BryantLam) [inline content/shmem_iget.tex]: Consistency: "When calling from" --> "When using"

**Comment by j-mike-culhane** [comment]
The purpose of this file is to remove usage of 'you' from the OpenSHMEM document. If I am to make consistency changes regarding the usage of 'using' over 'calling from', I would prefer to upload them in a dedicated using_fixes file. 

On that note, I prefer 'calling from' over 'using' as 'calling from' is more specific.

Can I get some additional thoughts on this? Who prefers 'calling from' and who prefers 'using'?

**Comment by jdinan** [comment]
I think's @BryantLam was suggesting something along the lines of ```When using the \Fortran bindings, ...```

**Comment by BryantLam** [comment]
Right. My suggestions were more intended to be consistent with the other changes that were already being introduced with this PR. It is possible that further changes are still needed to normalize the rest of the text, but I didn't hunt those out since, as you mentioned, it was out of scope.

**Comment by BryantLam** [comment]
If consensus is that my suggested changes should be in a different PR, I'm okay with that. The current changes are good with me.

**Comment by jdinan** [comment]
I think Bryant is right; the majority of cases say "using".  I'd prefer for this PR to also replace the handful of instances of "calling from" with "using" for consistency.

**Comment by j-mike-culhane** [comment]
Very well. For the sake of consistency, the suggested 'using' edits have been made.

**Comment by jdinan** [comment]
Thanks, @j-mike-culhane!  Could we get another review of the updated PR?

**Comment by manjugv** [comment]
👍
